packages/svelte/test/performance.test.ts

```diff
@@ -3,25 +3,26 @@ import { act, render } from '@testing-library/svelte';
 
 // linter doesn't like Svelte component imports
 import DummyComponent from './components/Dummy.svelte';
+import { vi } from 'vitest';
 
 let returnUndefinedTransaction = false;
 
 const testTransaction: { spans: any[]; startChild: jest.Mock; end: jest.Mock; isRecording: () => boolean } = {
   spans: [],
-  startChild: jest.fn(),
-  end: jest.fn(),
+  startChild: vi.fn(),
+  end: vi.fn(),
   isRecording: () => true,
 };
-const testUpdateSpan = { end: jest.fn() };
+const testUpdateSpan = { end: vi.fn() };
 const testInitSpan: any = {
   transaction: testTransaction,
-  end: jest.fn(),
-  startChild: jest.fn(),
+  end: vi.fn(),
+  startChild: vi.fn(),
   isRecording: () => true,
 };
 
-jest.mock('@sentry/core', () => {
-  const original = jest.requireActual('@sentry/core');
+vi.mock('@sentry/core', () => {
+  const original = vi.importActual('@sentry/core');
```

Comment:
iirc this needs to be `await`ed, right? Might explain the test fail.

Reply:
added await, it was also documented as promise and should be awaited I just didn't catch, thank!